00-HTML-CSS-basics/ex1.VI.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Ex 1.VI</title>
+		<meta charset="UTF-8">
+		<meta name="author" content="Martin Kalaizich">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <form>
+            <Label>First name:</Label><br><input type="text" name="firstname" required><br>
```

Comment:
Remember: `Label` is not the same as `label`. A lot of these things are case-sensitive.